src/terra-core/TerraCoreDataModel.ttl

```diff
@@ -2791,11 +2737,11 @@ TerraCore:hasSequenceLocation
   rdfs:domain TerraCore:VariantCall ;
   rdfs:label "hasSequenceLocation" ;
   rdfs:range [
-    owl:unionOf (
-      xsd:string 
-      TerraCore:SequenceLocation 
-    );
-  ];
+      owl:unionOf (
+          xsd:string
```

Comment:
Can't unionOf object with datatype

Reply:
Deferred as there are many of these - See[ DM-507](https://broadworkbench.atlassian.net/browse/DM-507)